DOCSP-41456: Rename Realm and clarify terminology on Device Sync landing page (#813)
mongodb/docs-app-services: source/sync/get-started.txt

   :depth: 2
   :class: singlecol

To sync data across devices, :ref:`enable Device Sync <enable-sync>` for your

> App Services App, then use the sync-related methods and properties in the

Author: Excellent callout! I've captured this work in [DOCSP-41847](https://jira.mongodb.org/browse/DOCSP-41847)

.. note::

>    The Web SDK does not currently support Realm Database or Atlas Device Sync.

Reviewer: I don't think this is true anymore, and it never got updated. The Web SDK has a Device Sync Preview, and has had for some time: https://www.mongodb.com/docs/atlas/device-sdks/web/sync/

Author: thank you!

   However, you can query the same data in an Atlas cluster using
   :ref:`MongoDB Data Access <web-mongodb-data-access>`.

Define Data Model
~~~~~~~~~~~~~~~~~

A synced database uses object models that you define to determine the type of
objects in the database. Additionally, Sync requires a server-side document schema

> for each synced object type so that App Services can translate and apply changes

Reviewer: We may want to remove "App Services" from here since it's now possible to use Device Sync without going through App Services.
```suggestion
for each synced object type so the server can translate and apply changes
```

  object models and code that you use in your production app. This is useful if
  you prefer a client-first approach that uses idiomatic object models in your
  preferred programming language.

  To get started, check out :ref:`create-sync-schema`.

> - **Generate object models from a document schema:** App Services can automatically

Reviewer: I'm not sure "App Services" is a hard requirement here, either. I _assume_ that if I enable Device Sync from Data Services, I can still generate models without going through App Services, but I'm not sure - I haven't tried it.

Maybe we should generally scrub these two pages from "App Services" mentions - the more I look at this content, the more I think we need to genericize it to reflect the multiple entry points to Device Sync.

Define Data Access Rules
~~~~~~~~~~~~~~~~~~~~~~~~

Data access rules determine which data to sync, as well as each user's
ability to read and write data. Rules are closely linked to your app's data
model.

> With Flexible Sync, you specify which data to sync through queries for matching

Reviewer: Just a nitpick - I think we can remove "Flexible" from "Sync" on these two pages, since it's no longer possible to make PBS apps (generally, except in weird edge cases) and we're not exposing the Sync Mode toggle anymore.

Author: Good catch!
